### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by peluprvi**
## Description

New README areas and updated sintaxe.
Since it is a suggestion, please feel free to edit it.

## Motivation and Context

Create new sections and some README makeover.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have created a PR in the [documentation](https://github.com/vuetifyjs/vuetifyjs.com) with the necessary changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,21 +42,37 @@
   </a>
 </p>
 
-<h2>Introduction</h2>
+<br>
+
+## Status
```
Comment (nekosaur) [inline README.md]: I would put the badges under then vuetify logo at the top. It is the de-facto standard way of displaying them in readmes.

File: README.md [hunk 2]
```diff
@@ -72,21 +42,37 @@
...
-  
-<p>Vuetify supports all <strong>modern browsers</strong>, including IE11 and Safari 9+. From mobile to laptop to desktop, you can rest assured that your application will work as expected. Interested in the bleeding edge? Try the vue-cli Webpack SSR (Server side rendered) template and build websites optimized for SEO.</p>
+Build *amazing* applications with the power of Vue and Material Design and a massive library of beautifully crafted components. Created according to Google's **<a href="https://material.io/" target="_blank">Material Design Spec</a>**, Vuetify components feature an easy-to-remember semantic design that shifts remembering complex classes and markup, to type-as-you speak properties that have simple and clear names.
 
-<h2>Demo and Documentation</h2>
+Harness the power of the [Vuetify community](https://chat.vuetifyjs.com) and get help 24/7 from talented developers across the world, the dev team and the creator, John Leider. Become a [backer](https://www.patreon.com/vuetify) and get access to dedicated support from the team.
```
Comment (nekosaur) [inline README.md]: Suggestion:

Harness the power of the [Vuetify community](https://chat.vuetifyjs.com) and get help 24/7 from the development team and our talented community members across the world.

File: README.md [hunk 3]
```diff
@@ -72,21 +42,37 @@
...
 
-<a href="https://vuetifyjs.com" target="_blank">Documentation</a>
+### Browser Support
 
-<h2>One minute Quick-start</h2>
+Vuetify supports all **modern browsers**, including IE11 and Safari 9+. From mobile to laptop to desktop, you can rest assured that your application will work as expected. Interested in the bleeding edge? Try the vue-cli Webpack SSR (Server side rendered) template and build websites optimized for SEO.
```
Comment (nekosaur) [inline README.md]: Vuetify supports all **modern browsers**, including IE11 and Safari 9+ (using polyfills).

File: README.md [hunk 4]
```diff
@@ -129,7 +115,7 @@ npm install vuetify
 yarn add vuetify
 ```
 
-## Use
+## Usage
 
 ```javascript
 import Vue from 'vue'
```
Comment (nekosaur) [inline README.md]: Add 

```
require('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons')
```

to the webpack part

File: README.md (hunk 1 above)
Comment (KaelWD) [reply README.md]: That's how it is currently, I had them nicely laid out.

File: README.md (hunk 2 above)
Comment (peluprvi) [reply README.md]: I updated the first phrase as your suggestion and I kept the second one.

File: README.md (hunk 4 above)
Comment (peluprvi) [reply README.md]: Done

**Pull request merged**